nexus/src/external_api/http_entrypoints.rs

```diff
@@ -305,7 +313,11 @@ async fn organizations_get_organization(
     apictx.external_latencies.instrument_dropshot_handler(&rqctx, handler).await
 }
 
-/// Delete a specific organization.
+/// Delete an organization
+///
+/// Permanently deletes an organization. It cannot be undone. This will not remove any
```

Comment:
I think we want that behavior for orgs, really the only one I want the behavior of nuke the world is for projects. Since test projects are going to be very common.

Reply:
Happy to put that on the list of "MVP" things ... but either way, it's not what the code does today and I think it's better to have the docs reflect reality and update them when we change things.

(In terms of doing this: to do an async removal correctly, I think we'd need to (1) mark the project deleted so that you can't create new things in it, (2) go crawl the thing and remove everything in it, which can take arbitrarily long and get interrupted, so presumably this has to happen in a saga, (3) go remove the Project itself.  It's an open question (to me) whether we'd let you access things in the Project while that's happening.  And what if we run into something we can't delete, like a running Instance?  These are all the reasons I'd love to punt if we can.)

Reply:
oh for sure ill make this match reality sorry I've just been doing other stuff will update this

yeah mostly what I was saying is in testing we likely will script this anyways so might be worth doing for reals, but also not to bad to script through the API , I did similar in the cli repo (minus sagas of course but just recursively calling API endpoints to delete) for the cleanup of tests)

Reply:
In azure, one of the things i really liked was that deleting the higher level folder (ie projects) would delete everything even running instances from rfd 4 https://github.com/oxidecomputer/rfd/tree/master/rfd/0004/#22-projects